pkg/importer/util.go

```diff
@@ -41,7 +42,9 @@ func MoveFile(src, dst string) error {
 // StreamDataToFile provides a function to stream the specified io.Reader to the specified local file
 func StreamDataToFile(dataReader io.Reader, filePath string) error {
 	// Attempt to create the file with name filePath.  If it exists, fail.
-	outFile, err := os.OpenFile(filePath, os.O_CREATE|os.O_EXCL|os.O_WRONLY, os.ModePerm)
+	var outFile *os.File
```

Comment:
The original way this was written is the standard go way.

Reply:
yeah, this is leftover was a previous change and revert